src/api/java/baritone/api/utils/ExampleBaritoneControl.java

```diff
@@ -295,6 +295,10 @@ public boolean runCommand(String msg0) { // you may think this can be private, b
             logDirect(success ? "Loaded" : "Unable to load");
             return true;
         }
+        if (msg.startsWith("schematica")) {
```

Comment:
detect if they have schematica and add to help command 🤔

Reply:
maybe "secret" command

Reply:
this